easybuild/tools/module_naming_scheme/hierarchical_mns.py

```diff
@@ -189,6 +192,9 @@ def det_modpath_extensions(self, ec):
 
                         if non_dummy_tc:
                             tc_comp_name, tc_comp_ver = tc_comp_info
+                            # Stick to name GCC for GCCcore
+                            if tc_comp_name == GCCCORE:
+                                tc_comp_name = 'GCC'
```

Comment:
Are you sure this won't cause things built with GCCcore to end up in .../Compiler/GCC instead of Compiler/GCCcore

I'm not sure exactly what this affects.

Reply:
No, it won't affect that since GCCcore is built with the dummy compiler (so the condition `non_dummy_tc` 4 lines up is not satisfied). This is only relevant to compilers built with GCCcore and only affects the searching for matches in `COMP_NAME_VERSION_TEMPLATES` (see the for loop around this block).

I'm not overly confident that this has no impact though but I don't believe it does. As far as I know Clang is the only compiler build from source using GCCcore (rather than GCC).

Reply:
I just tried this out with Clang, and then with a normal package
`eb Ghostscript-9.27-GCCcore-8.2.0.eb --include-module-naming-schemes hierarchical_mns.py`
and it was placed correctly (under `software/Compiler/GCCcore/Ghostscript` )